sevntu-checks/pom.xml

```diff
@@ -283,6 +294,8 @@
               <regex><pattern>.*.checks.coding.MapIterationInForEachLoopCheck</pattern><branchRate>90</branchRate><lineRate>98</lineRate></regex>
               <regex><pattern>.*.checks.coding.NoNullForCollectionReturnCheck</pattern><branchRate>85</branchRate><lineRate>96</lineRate></regex>
               <regex><pattern>.*.checks.coding.OverridableMethodInConstructorCheck</pattern><branchRate>94</branchRate><lineRate>99</lineRate></regex>
+              <regex><pattern>.*.checks.coding.Jsr305AnnotationsCheck</pattern><branchRate>84</branchRate><lineRate>93</lineRate></regex>
+              <regex><pattern>.*.checks.coding.Jsr305AnnotationsCheck.AbstractJsr305Check</pattern><branchRate>95</branchRate><lineRate>98</lineRate></regex>
```

Comment:
Please also provide a code coverage report. We would prefer 100%.

Reply:
See the attached report: ![jacoco zip](https://user-images.githubusercontent.com/383271/56152594-6c76fc80-5fb4-11e9-8668-c508f25ded6b.jpg)

(please rename file to `jacoco.zip`, github for whatever reason would not accept the zip extension)

Reply:
The coverage report can be viewed online here: https://mbert.github.io/sevntu-checkstyle/coverage/jacoco/com.github.sevntu.checkstyle.checks.coding/Jsr305AnnotationsCheck.java.html